code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -31,17 +26,45 @@
 	tod = 0
 	timeofdeath = 0
 
+/mob/living/carbon/human/adjustBrainLoss(var/amount)

```

Comment:
Hm, does this also mean that mannitol will now cure physical brain damage, too, or will that still be surgery only?